api/src/api/emailCheckRouter.ts

```diff
@@ -0,0 +1,34 @@
+import { UserDataClient } from "@domain/types";
+import type { LogWriterType } from "@libs/logWriter";
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+
+export const emailCheckRouterFactory = (userDataClient: UserDataClient, logWriter: LogWriterType): Router => {
```

Comment:
[pebble] How would y'all feel about moving this under `userRouterFactory`? We have a number of other endpoints that deal with getting or updating user data in that factory and this feels like an extension of that existing logic.

Reply:
This might inadvertently fix your lint issue.